packages/config/src/config.js

```diff
@@ -16,12 +17,12 @@ module.exports = ({
         optimization: {
             minimize: (process.env.NODE_ENV || mode) === 'production',
             splitChunks: {
+                chunks: 'all',
+                maxInitialRequests: Infinity,
```

Comment:
I think that there should be some upper limit for the entry point requests. Usually is set around 20-40

Reply:
I went with example of how to do it from this [blog](https://medium.com/hackernoon/the-100-correct-way-to-split-your-chunks-with-webpack-f8a9df5b7758). I might add a few more `cacheGroups` though. So we have at least PF, FEC and react dependencies in seperate files.